dashboard/static/styles/main.styl

```diff
@@ -26,6 +26,11 @@ section {
     background-position: top center;
 }
 
+.top-bar-title h1 {
+    font-size: 1rem;
+    font-weight: normal;
```

Comment:
We can leave this out, since `normal` is the default. Or did you mean to copy `font-weight: bold` from below?